.github/workflows/go.yml

```diff
@@ -5,6 +5,10 @@ on:
     paths:
       - '**.go'
 
+permissions:
+  contents: write
+  pages: write
+
```

Comment:
what are these permissions for? I see they are required for `ncruces/go-coverage-report` in the other workflow

Reply:
Excellent catch! I created the second workflow specifically so that the go.yml wouldn't get these permissions.

Reply:
Fixed!